corehq/apps/userreports/transforms/custom/date.py

```diff
@@ -1,5 +1,13 @@
 import calendar
+from datetime import datetime
+from dimagi.utils.dates import force_to_datetime
 
 
 def get_month_display(month_index):
     return calendar.month_name[int(month_index)]
+
+
+def days_elapsed_from_date(date):
+    date = force_to_datetime(date)
+    now = datetime.utcnow()
+    return (now - date).days
```

Comment:
@czue I think that this problem only occurs in date sources but I want to use this in report. Frankly, I thought that transform is only defined for reports. I agree that it doesn't make sense for data sources. 

Maybe I wasn't precise what I want to achieve. Basically I want to display in report how many days elapsed from some date stored in defined UCR data source.